### PR TITLE
Add Pie Chart Feature

### LINKED ISSUE #83: New feature: Pi Chart visualisation of number of students

### PR DESCRIPTION
**Pull request by hooyushing**
fixes #83

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/PiChartCommand.java
Comment (sumomomomomo) [inline src/main/java/seedu/address/logic/commands/PiChartCommand.java:1]: Should be named `PieChartCommand`, not `PiChartCommand`

File: src/main/java/seedu/address/logic/commands/PiChartCommand.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    @Override
+    public CommandResult execute(Model model) {
+        requireNonNull(model);
+        Map<String, Integer> numOfStudentsInEachClass = new HashMap<>();
+        model.getFilteredPersonList().forEach(person -> {
+            String classId = person.getClassId().value;
+            numOfStudentsInEachClass.put(classId, numOfStudentsInEachClass.getOrDefault(classId, 0) + 1);
+        });
+        assert numOfStudentsInEachClass != null : "numOfStudentsInEachClass map should not be null";
```
Comment (sumomomomomo) [inline src/main/java/seedu/address/logic/commands/PiChartCommand.java:27]: nit: this assert will never fail because you create a HashMap object in line 22

**Review by sumomomomomo — COMMENTED** [review]
Overall good, but I think you should rename every instance of "pi" to "pie"

File: src/main/java/seedu/address/logic/commands/PiChartCommand.java (hunk 1 above)
Comment (hooyushing) [reply src/main/java/seedu/address/logic/commands/PiChartCommand.java:27]: lol

File: src/main/java/seedu/address/logic/commands/PiChartCommand.java
Comment (hooyushing) [reply src/main/java/seedu/address/logic/commands/PiChartCommand.java:1]: done

**Comment by sumomomomomo** [comment]
LGTM

**Pull request merged**